### PR TITLE
Documentation for model interdependencies

### PR DESCRIPTION
**Pull request by validbeck**
## Internal Notes for Reviewers

> For sc-5863 I created a net-new page (Configure model interdependencies) and a net-new section (Managing the model inventory).

### Managing the model inventory
- Net-new page: `model-inventory/configure-model-interdependencies.qmd`
- Moved `model-inventory/manage-inventory-custom-fields.qmd` under here as well
- I will likely also put our archive/delete models page here when that feature is ready

| New sub-section/landing | Adjusted "Guides" listing |
|---|---|
| Screenshot 2024-08-19 at 11 52 03 AM | Screenshot 2024-08-19 at 11 52 08 AM |

### Configure model interdependencies
Net-new page, explaining what model interdependencies are within ValidMind, and how to view and work with them.

Screenshot 2024-08-19 at 11 56 00 AM
Screenshot 2024-08-19 at 11 54 14 AM
Screenshot 2024-08-19 at 11 54 30 AM

### CODE REVIEW

**Comment by validbeck** [comment]
@gtagle & @panchicore Nik suggested that for feature work I start tagging in the feature owner to confirm if the documentation is accurate if he can't get to it fast enough! Let me know if you two can help/have any questions. :)

File: site/guide/model-inventory/configure-model-interdependencies.qmd [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+:::
+
+## View interdependencies
+
+1. In the left sidebar, click **Model Inventory**. 
+
+2. Select a model by clicking on it or find your model by applying a filter or searching for it.[^3]
+
+3. Any existing model interdepencies will be shown under the Model Interdepencies section.
```
Comment (nrichers) [inline site/guide/model-inventory/configure-model-interdependencies.qmd:27]: Use present tense and spell good: 😛 

```suggestion
3. Any existing model interdependencies are shown under the Model Interdependencies section.
```

File: site/guide/model-inventory/configure-model-interdependencies.qmd [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    You can swap between the **{{< fa arrow-up >}} Upstream** or **{{< fa arrow-down >}}Downstream** tabs to set both interdependency types at once.
+
+5. On your list of models populated by the model inventory, add or remove interdependencies as desired: 
+
+    - **Add** — Check off any models that are either upstream or downstream dependent on this current model.
+    - **Remove** — Uncheck any models previously linked as upstream or downstream dependent on this current model.
+
+6. Click **Save** to apply your changes.
+
+Once added, model interdepencies will populate on all models linked together.
```
Comment (nrichers) [inline site/guide/model-inventory/configure-model-interdependencies.qmd:58]: Use the same action word for consistency and spell good: 

```suggestion
Once applied, model interdependencies will populate on all models linked together.
```

**Review by nrichers — COMMENTED** [review]
Thank you for addressing these docs that were missed during our product development cycle! Yes, let's start asking the originators for these changes to review the docs for accuracy. My comments are more of an add-on from the docs perspective.

**One item to discuss:** I’m a bit confused by what should go under “Working with the model inventory” and “Managing the model inventory” in these changes. There’s overlap between topics in these sections — I’d maybe get rid of one of these? 

For example, you could have “Register models” as a top-level topic, followed by “Managing the model inventory” with the all the other topics under it, including those from “Working with the model inventory”, and then get rid of that “Working with …” heading.

**Comment by validbeck** [comment]
> One item to discuss: I’m a bit confused by what should go under “Working with the model inventory” and “Managing the model inventory” in these PR changes. There’s overlap between topics in these sections — I’d maybe get rid of one of these?

To me, the existing working with the model inventory section is more of a "Get started" sort of idea, and the "Managing" is more advanced set up. 🤔 I personally like the separation of these two ideas, but I'm in agreement that maybe the seconds can be renamed? I know that the "Working with..." topic is linked A LOT elsewhere, including the `developer-framework` notebooks, so it might be a PITA to rename though.

**Comment by validbeck** [comment]
How's this? 🤔 

Screenshot 2024-08-21 at 9 31 36 AM

**Review by nrichers — APPROVED** [review]
> To me, the existing working with the model inventory section is more of a "Get started" sort of idea, and the "Managing" is more advanced set up. 🤔 

It looks like the short descriptions now explain this better after your last commit. LGTM, though we might simply want to name the first topic "Getting started with ..." at some point. :shipit:

**Pull request merged**